doc/experiment.rst

```diff
@@ -1,327 +1,423 @@
+     - ``exp.start(*args[, block, summary, ...])``
+     - Launch an Entity
+   * - ``stop()``
+     - ``exp.stop(*args)``
+     - Clobber an Entity
+   * - ``get_status()``
+     - ``exp.get_status(*args)``
+     - Retrieve Entity Status
+
+Orchestrator
+------------
+The orchestrator is an in-memory database with features designed
+to enable a wide variety of AI-enabled workflows, including features
+for online training, low-latency inference, cross-application data
+exchange, online interactive visualization, online data analysis, computational
+steering, and more. The ``Orchestrator`` can be thought of as a general
+feature store capable of storing numerical data, ML models, and scripts
+and capable of performing inference and script evaluation on feature
+store data. Any SmartSim ``Model`` or ``Ensemble`` model can connect to the
+``Orchestrator`` via the :ref:`SmartRedis<SmartRedis Client Library Hook>`
+client library to transmit data, execute ML models, and execute scripts.
+
+**SmartSim offers two types Orchestrator deployments:**
+
+* :ref:`Clustered Orchestrator <Clustered Orchestrator>`
+* :ref:`Colocated Orchestrator <Colocated Orchestrator>`
+
+Clustered Orchestrator
+^^^^^^^^^^^^^^^^^^^^^^
+The orchestrator can be composed of one or more in-memory database shards that are spread
+across one or more compute nodes.
+The multiple compute hosts memory can be used together to store data.
+Users do not need to know how the data is stored in a clustered
+configuration and can address the cluster with a SmartRedis client
+like a single block of memory using simple put/get semantics in SmartRedis.
+The database shards communicate with each other via TCP/IP in the driver script and application.
+SmartRedis will ensure that data is evenly distributed among all nodes in the cluster.
+
+Clustered Deployment Diagram
+""""""""""""""""""""""""""""
+During clustered deployment, a SmartSim ``Model`` (the application) runs on separate
+compute node(s) from the database node(s).
+A clustered database is optimal for high data throughput scenarios
+such as online analysis, training and processing.
+
+Below is an image illustrating communication
+between a clustered orchestrator and a
+multi-node model. In the Diagram, an instance of the application is
+running on each application compute node. A single SmartRedis client is initialized with
+the clustered database address and used to communicate with the application's compute nodes.
+Data is streamed from the application compute nodes to the sharded database via the client.
+
+.. figure::  images/clustered-orc-diagram.png
+
+Initialize a Clustered Orchestrator
+"""""""""""""""""""""""""""""""""""
+To create an ``Orchestrator`` that does not share compute resources with other
+SmartSim entities, use the ``Experiment.create_database()`` factory method.
+Specifying the parameter `db_nodes` as greater than or equal to 1 will determine
+whether your database is multi-sharded or single-sharded.
```

Comment:
This sentence is unintentionally ambiguous (shards >= 1). Suggest rewriting as:
Specifying 1 for the `db_nodes` parameter causes the database to be single-sharded; otherwise it is multi-shared.